_docs/concepts/configuration/model.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Data model
+overview: Istio configuration data model.
+
+order: 10
+
+layout: docs
+type: markdown
+---
+
+Istio config is a conglomerate of config aspects. Each config aspects is
```

Comment:
I'm worried that this text is generally too focused on implementation strategy, as opposed to the user experience. Consider https://kubernetes.io/docs/concepts/overview/kubernetes-api/ as an example that's more focused on what the user sees rather than how it's implemented underneath. Can we adapt their content and just drop it here instead?

Reply:
+1, fully agree. Anything on istio.io needs to be for the end user of Istio, and needs to be kept as simple as possible.